### DISCUSSION TITLE
How to SSH into my TPOT that is running on google cloud

### DISCUSSION THREAD

**Discussion by AxiomaticTruth** [telekom-security/tpotce#1308] [Q&A] [answered]
Hello I am trying to run TPOT to collect data for a university assignment we were asked to use Google Cloud but I can't connect to the SSH through port 64295 (edited was from 64297 to ...95 as I typed it incorrectly).

My dashboard stopped responding and appeared to crash completely so I manually rebooted the cloud VM and waited to see if the dashboard would come back up which it didn't/hasn't so I then tried to connect to the SSH to try and fix this but I can't get it working and was wondering if anyone could help.

I either get an error saying "[(myuser)]@(myip): Permission denied (publickey)" when I try and SSH in using Linux machine or if I use the SSH connection through Google Cloud "Code: 4010 Reason: destination read failed" (I have tried to do their firewall suggested fix it doesn't work).

Any help would be hugely appreciated as I really don't want to fail my coursework.

thanks

**Comment by I-Simon-I** [comment]
Hi ! 

I'm no expert, but don't you have to use port 64295 to access T-Pot via SSH ? As mentionned here https://github.com/telekom-security/tpotce#ssh-and-cockpit

Don't know if it will help

**Reply by AxiomaticTruth** [reply]
Sorry yes you are correct I had a long night when I posted this when trying to connect to the SSH I was using the port 64295 but I still received the "Permission denied (publickey)" response.

Thank you for your time

**Comment by G0nxo** [comment, marked as answer]
Hello, I don't know if you still have this problem. I've been struggling for half an hour to connect to the instance via SSH in Google Cloud and I found the solution.
You have to go into the details of your instance, and in the upper part you have the SSH button. You just have to clic on the arrow and select "connect to the specified port" and there you introduce 64295. 
This is how it worked for me, because while trying to connect via SSH I got the same publickey error.
